scene/3d/voxel_gi.cpp

```diff
@@ -273,14 +273,14 @@ VoxelGI::Subdiv VoxelGI::get_subdiv() const {
 	return subdiv;
 }
 
-void VoxelGI::set_extents(const Vector3 &p_extents) {
-	// Prevent very small extents as these break baking if other extents are set very high.
-	extents = Vector3(MAX(1.0, p_extents.x), MAX(1.0, p_extents.y), MAX(1.0, p_extents.z));
+void VoxelGI::set_size(const Vector3 &p_size) {
+	// Prevent very small size dimensions as these breaks baking if other size dimensions are set very high.
+	size = Vector3(MAX(1.0, p_size.x), MAX(1.0, p_size.y), MAX(1.0, p_size.z));
```

Comment:
I'm skeptical about this. This effectively decreases the minimum size by half

Reply:
Yeah that was my thought as well. So we should probably use `0.5` here, right?

Reply:
Okay, should be fine. If I recall correctly the minimum size was chosen rather arbitrarily